src/content/basics/aws-architecture/index.md

```diff
@@ -5,7 +5,24 @@ date: 2020-06-15
 weight: 50
 type: page
 categories: ["basics"]
-last-review-date: 2020-05-18
+last-review-date: 2020-10-28
+user_questions:
+  - What areas are covered in the Giant Swarm AWS platform?
+  - Do I need a control plane per AWS region?
+  - What does Giant Swarm use operators for?
+  - How does Ginat Swarm Manage Custom Resources (CRs)?
+  - How can users of Giant Swarm manage their clsuters?
```

Comment:
To me this question appears too broad. I'd remove it.

Reply:
Are you referring to the question on line 14?

Reply:
Yes